extensions/adobe/experience/workfront/categoryparameter.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/workfront/categoryparameter",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Work Category Attributes",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/workfront/object",
+    "https://ns.adobe.com/experience/workfront/changeevent"
+  ],
+  "description": "Category Parameter related fields",
+  "definitions": {
+    "wfobject": {
```

Comment:
Not an issue but I would suggest to change the definition name here to something like "category-parameter" . I guess it's a copy paste from the previous field group.

Reply:
I made this change, can you provide a little more detail of where this field/attribute is being used?